Resources/Prototypes/Entities/Objects/Weapons/Guns/pneumatic_cannon.yml

```diff
@@ -0,0 +1,25 @@
+﻿- type: entity
+  name: improvised pneumatic cannon
+  parent: BaseItem
+  id: ImprovisedPneumaticCannon
+  description: Is this even legal?
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/pneumatic_cannon.rsi
+    netsync: false
+    layers:
+    - state: pneumaticCannon
+      map: [ "enum.PneumaticCannonVisualLayers.Base" ]
+    - state: oxygen
+      map: [ "enum.PneumaticCannonVisualLayers.Tank" ]
+      visible: false
+  - type: Item
+    size: 24
```

Comment:
I can't help but notice that the cannon has 30 capacity but 24 size. Hrm...

Reply:
![image](https://user-images.githubusercontent.com/19853115/132157713-7f33746c-9007-4d52-a457-5f82d27df1e2.png)